docs/udf.rst

```diff
@@ -402,6 +402,34 @@ An example code can be found `here <https://github.com/Open-EO/openeo-python-cli
 
 
 
+Logging from the UDFs
+==============================
+
+In some cases, users might want to use print statements within a UDF so that they can monitor their 
```

Comment:
'users might want to' => I would address the user more directly:
In some cases, you may want to log information from your user defined function, for instance to provide debug information or to log warnings.